src/components/vm/VmInitializer.tsx

```diff
@@ -60,6 +61,7 @@ export default function VmInitializer() {
   const setVmStore = useVmStore((store) => store.set);
   const { requestAuthentication, saveCurrentUrl } = useSignInRedirect();
   const idOS = useIdOS();
+  const router = useRouter();
```

Comment:
Kindly remove this and its import if it's not necessary.

Reply:
fixed